### ISSUE TITLE
Fail to npm build action

### ISSUE THREAD

**Issue by PabloCarreira** [mediamonks/display-ads-recorder#11]
https://github.com/mediamonks/display-ads-recorder/blob/179dee4c40903cf86f8bc4ad96b6a866c3f195e6/.github/workflows/Publish-to-NPM.yml#L15

![image](https://github.com/mediamonks/display-ads-recorder/assets/18534827/d86a1034-c043-40e3-8f99-762ae44ad7ed)

Looks like we need to update the node version usage

**Comment by mm-paulie** [comment]
Yes I've tried to fix this in the past but then it was for some reason not working, not sure why, it just looks like you can fix it by updating the node version

**Comment by PabloCarreira** [comment]
@mientjan can you Please Update this workflow file and just update node version? I don't have npm token permission.

**Comment by mientjan** [comment]
what npm version?

**Comment by xaviergdiez84** [comment]
we should have 11.6.3 on NPM, currently is 11.6.2:

[image: image.png]
Kind regards,
Met vriendelijke groet,
Xavier Garcia (he/him/his )
Digital Advertising Technical Director
monks.com
<https://monks.com/> <https://media.monks.com/> <https://media.monks.com/>
